src/SIL.LCModel.Core/WritingSystems/CoreWritingSystemDefinition.cs

```diff
@@ -67,14 +67,14 @@ private void MainCharacterSetChanged(object sender, NotifyCollectionChangedEvent
 			switch (e.Action)
 			{
 				case NotifyCollectionChangedAction.Add:
-					AddWordFormingOverrides(e.NewItems.Cast<string>());
+					AddWordFormingOverrides(e.NewItems!.Cast<string>());
```

Comment:
these can be null so we should just throw manually if it is

Reply:
I'm not sure why I added these. When I remove them, there are no warnings. I get that they can be null, based on the spec, but I'm not sure they could ever cause a null exception in their respective case statements.

Reply:
Ok, `dotnet build` produces a warning, but the IDE doesn't. That's why they were there. I'll add the explicit checks